packages/fxa-react/styles/inputs.css

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* When we use this in React and no longer need it in content-server, we should use
+ * a component instead of this class abstraction if possible. */
+.input-radio {
+  @apply absolute appearance-none w-full h-full top-0 drop-shadow-sm bg-grey-10 rounded-lg hover:border-grey-100 border-transparent border-2 focus-visible:outline-none;
+
+  &:checked {
+    @apply bg-gradient-to-tr from-blue-600/10 to-purple-500/10 border-blue-500;
+  }
+
+  &:not(:checked) {
+    @apply focus:shadow-input-blue-focus focus-visible:shadow-input-blue-focus;
```

Comment:
Might be nice to make the focus indicator visible whether checked or unchecked - when I keyboard into the radio button fieldset I don't see what is in focus.

In another a11y observation, I noticed that when I keyboard into the inputs, I can't select the first option by clicking enter or tabbing to the button. I have to arrow key to option 2 then arrow back to select the first option.

Reply:
Correct me if I'm wrong but I think you're seeing the default behavior of radio buttons, which I say because I thought the same until I played around with it, see it here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

I do think I'm going to need a quick a11y follow up though, there's some suggestions in Figma especially around focus state that I did not get to.